docs/basic-setup.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 4
+---
+
+# Basic Setup
+
+Follow the steps below to install OpenCost Open Source.
+
+See this page for all [Kubecost install options](http://docs.kubecost.com/install).
+
+## Prerequisites
+
+Install Prometheus:
+
+```sh
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm install my-prometheus prometheus-community/prometheus --namespace prom --create-namespace -f https://raw.githubusercontent.com/kubecost/opencost/develop/kubernetes/prometheus/extraScrapeConfigs.yaml
+```
+
+## OpenCost
+
+If providing your own Prometheus, set [this environment variable](https://github.com/kubecost/opencost/blob/c211fbc1244a9da9667c7180a9e4c7f988d7978a/kubernetes/deployment.yaml#L33) to the address of your prometheus server.
+
+If you used the Prometheus install command from above, this will install OpenCost on your cluster:
+
+```sh
+kubectl create namespace opencost
+kubectl apply --namespace opencost -f https://raw.githubusercontent.com/kubecost/opencost/develop/kubernetes/service-account.yaml
```

Comment:
Is it possible to apply everything in the folder with a single command like we previously did?

Reply:
unless I'm missing something, no. You'd have to clone the repo or apply these all separately. 
That said, I'm happy to make this an all in one yaml like we have for exporter.

Reply:
Here is the one-liner:
https://github.com/kubecost/opencost/blob/1ae724e3f30b2046f8c5326c3e640aa10406bd45/kubernetes/opencost.yaml

Please see new commit.

Reply:
We should be able to follow the same process covered in this [doc.](https://github.com/kubecost/opencost/blob/develop/deploying-as-a-pod.md)

It will recurse through the folder and apply all manifests.
`kubectl apply -f kubernetes/ --namespace cost-model`
Yes, the repo would need to be cloned locally. That is typically standard practice.

Reply:
I like the one-liner jesse created. Definitely simplifies IMO... have objections Sean?

Reply:
Nope!